crates/bevy_core_pipeline/src/main_pass_3d.rs

```diff
@@ -139,7 +139,7 @@ impl Node for MainPass3dNode {
                     // transparent ones.
                     depth_ops: Some(Operations {
                         load: LoadOp::Load,
-                        store: false,
+                        store: true,
```

Comment:
Please update the preceding comment and explain why the depth buffer is being written to as it isn't obvious from this code alone, and Xcode will likely warn about it as wasted bandwidth usage so people may come here and want to change it. If there is a comment explaining why it is being written, then we can hopefully avoid some ping-ponging. :)

Reply:
Done. I also filed #4174 as a reminder to remove the workaround after the issue is resolved.